gammapy/obs/datastore.py

```diff
@@ -226,19 +226,18 @@ def filename(self, obs_id, filetype='events', abspath=True):
     def make_observation_table(self, selection=None):
         """Make an observation table, applying some selection.
 
-        TODO: implement a more flexible scheme to make box cuts
-        on any fields (including e.g. OBSID or TIME
-        Not sure what a simple yet powerful method to implement this is!?
+        Wrapper function for `~gammapy.obs.Observationtable.select_observations`.
 
         Parameters
         ----------
-        selection : TODO
-            TODO: describe me
+        selection : `~dict`
+            Dictionary with a few keywords for applying selection cuts.
+            TODO: add doc with allowed selection cuts!!! and link here!!!
```

Comment:
Is it possible to address this TODO quickly?
(if no, maybe making a new issue for yourself with a task list to keep track of such things would be helpful?)